Users/Install.rst

```diff
@@ -83,10 +83,82 @@ To install coala only (without any bears), you can do:
     **Ubuntu Users** - Do not use ``apt-get install coala``
     for installing coala as that is different software.
 
+Installing inside a pipenv
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pipenv is the most user friendly method of managing virtual environment and
+packages.
+You can look at it as a mix of ``pip`` and ``virtualenv``. So you don't have
+to manage them seperately for your projects.
+
+First, assuming you have installed python and pip earlier from
+this guide we'll now install ``pipenv`` by running the following command.
+
+::
+
+    $ pip install pipenv
+
+Now we create a python 3 environment for our project, move to your project
+directory and run the following command.
+
+::
+
+    $ pipenv --three
+
+The above command will create virtual environment isolated from your system
+global installation.
+
+    On **Unix based** systems (OS X and Linux), you can find the user base
+    binary directory for your pipenv by running
+
+    ::
+
+        $ python -m site --user-base
+
+    Result of the above command will be something similar to
+    ``/home/<Username>/.local``.
+    And on **Windows**, run the following command
+
+    ::
+
+        $ py -m site --user-site
+
+    Result of the above command will be something similar to
+    ``C:\Users\<Username>\AppData\Roaming\Python36\site-packages``.
+    You'll have to add this result after replacing ``site-packages``
+    with ``Scripts`` i.e.
+    ``C:\Users\<Username>\AppData\Roaming\Python36\Scripts``
+    to PATH from Control Panel. This is only when you don't have this PATH
+    already set.
```

Comment:
It would be much easier to follow if Windows and Unix had their own sections with headers instead of being in one big block quote. Block quotes can be a little tough to read at times b/c of the coloring.

Reply:
I'll make it look like the version above in `System Wide Installation`. Hope that'll look more adaptive.

Reply:
The block structure under the section for `System Wide Installation` seems better. I'll make sure to format this block according to the looks of that block.